mLRS/Common/hal/esp/rx-hal-generic-900-esp8285.h

```diff
@@ -65,7 +65,7 @@
 #define SX_DIO0                   4
 #define SX_DIO1                   5
 
-IRQHANDLER(IRAM_ATTR void SX_DIO_EXTI_IRQHandler(void);)
+IRQHANDLER(void IRAM_ATTR SX_DIO_EXTI_IRQHandler(void);)
```

Comment:
does it matter? if so, ensure it's equal in main code

Reply:
I don't think it matters but I am trying to standardise on this `void IRAM_ATTR SX_DIO_EXTI_IRQHandler(void);`. Eventully we'll use the FAST_RAM macro for these.

Reply:
GREAT!

Reply:
actually, if IRAM_ATTR void works, can I ask you to settle on this pattern?

this is because other attr may come in fron, and one in fact could just define IRQHANDLER properly, so don't need that attr to be explicitely specified for ISRs. This should be the most portable and clean approach.